src/bin/nickel.rs

```diff
@@ -19,6 +19,12 @@ struct Opt {
     #[structopt(short = "f", long)]
     #[structopt(parse(from_os_str))]
     file: Option<PathBuf>,
+
+    #[cfg(debug_assertions)]
+    /// Do not include the standard library in the scope
```

Comment:
```suggestion
    /// Do not include the standard library in the scope (for debugging purposes only)
```

Reply:
I tried to make an explicit, complete and short message like so:
```
Skip the standard library import, for debugging only, does not affect REPL
```